recipes/cppbenchmark/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+
+class CppBenchmark(ConanFile):
+    name = "cppbenchmark"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/chronoxor/CppBenchmark"
+    description = "Performance benchmark framework for C++ with nanoseconds measure precision."
+    topics = ("conan", "utils", "library", "benchmark")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False],
+               "shared": [True, False],
+               "tests": [True, False],
+               "pdbs": [True, False]}
+    default_options = {"fPIC": True,
+                       "shared": False,
+                       "tests": False,
```

Comment:
```suggestion
```

Remove tests, the idea here is packaging only, not running unit tests. Conan offers support for unit tests indeed, but it's a feature for internal/company recipes.

Reply:
fixed in 29610bc0